src/abi/fuse_abi.rs

```diff
@@ -197,6 +197,10 @@ const SUBMOUNTS: u64 = 0x800_0000;
 // Filesystem responsible for clearing security.capability xattr and setuid/setgid bits.
 const HANDLE_KILLPRIV_V2: u64 = 0x1000_0000;
 
+// In upstream kernel it's FUSE_SETXATTR_EXT, but in Anolis kernel 5.10 it's FD_PASSTHROUGH.
+// We will make it compitible when the fd passthrough patch is merged into the upstream.
+const FD_PASSTHROUGH: u64 = 0x2000_0000;
```

Comment:
Please land a compatible solution in openanolis kernel first so that we do not need to add more compatibility code in the future for it.

Reply:
As we mentioned before, Bit 29 in the Init flag will use the definition from the Linux kernel community, and FD_PASSTHROUGH will be maintained by the application itself.

Additionally, in order to support FD_PASSTHROUGH, the return value of create() and open() methods in the FileSystem trait needs to include an additional u32 variable. How should this be implemented - should a new return value be added or should it be included in the OpenOptions struct?
@eryugey @bergwolf

Reply:
In https://gitee.com/anolis/cloud-kernel/blob/devel-5.10/include/uapi/linux/fuse.h#L378, this is 
``` C
#define FUSE_PASSTHROUGH	(1ULL << 63)
```

Reply:
> In https://gitee.com/anolis/cloud-kernel/blob/devel-5.10/include/uapi/linux/fuse.h#L378, this is
> 
> ```c
> #define FUSE_PASSTHROUGH	(1ULL << 63)
> ```

Yes, this PR will resolve the problem: https://github.com/cloud-hypervisor/fuse-backend-rs/pull/110/commits/a9f2580cfa2658f97115620393919615dc893016

Reply:
Please fold it here so that we add the feature properly in one PR.

Reply:
OK~